pkg/config/config.go

```diff
@@ -98,7 +98,12 @@ func (c *Config) fillDefaults() error {
 		URL:             "https://localhost:6443",
 		Port:            6443,
 	}
-	c.ApiServer.AuditLog.Profile = "Default"
+	c.ApiServer.AuditLog = AuditLog{
+		MaxFileAge:  10,
```

Comment:
In https://github.com/openshift/openshift-docs/pull/75233, we say that "The `maxFileAge` default is 0, meaning there is no default age limit." - we should make sure that the docs agree with the value being set here.

Reply:
Fixed